internal-scripts/test-debs

```diff
@@ -280,6 +280,17 @@ if [[ "$VARIANT" = jemalloc ]]; then
     run c++ "-I$INCLUDE_DIR" -Wall /system/resources/test-env/jemalloc-cxx.cpp -o /tmp/jemalloc_cxx
 fi
 
+if [[ "$RUBY_PACKAGE_VERSION" =~ ^3\.2 ]]; then
```

Comment:
Rather than checking whether version is 3.2.x, the check should be version >= 3.2.

Reply:
Added at 8ee5879